man/mustache.5.ron

```diff
@@ -85,16 +92,113 @@ Output:
     * &lt;b&gt;GitHub&lt;/b&gt;
     * <b>GitHub</b>
 
+**Dotted Names**
+
+If the `name` contains dots, it is split on the dots to obtain multiple
+keys. The first key is looked up in the context as described above. If it
+is found, the next key is looked up within the previous result. This is
+repeated until a key is not found or until the last key is found. The
+final result is interpolated as above.
+
+Template:
+
+    * {{client.name}}
+    * {{age}}
+    * {{client.company.name}}
+    * {{{company.name}}}
+
+Hash:
+
+    {
+      "client": {
+        "name": "Chris & Friends",
+        "age": 50
+      },
+      "company": {
+        "name": "<b>GitHub</b>"
+      }
+    }
+
+Output:
+
+    * Chris &amp; Friends
+    *
+    *
+    * <b>GitHub</b>
+
+**Implicit Iterator**
+
+As a special case, if the `name` consists of only a dot and nothing else,
+the value that **is** the current context is interpolated as a whole. This
+is especially useful if the parent context is a list; see **Sections**
+below.
+
+Template:
+
+    * {{.}}
+
+Current context:
```

Comment:
This is somewhat unrelated to the PR, but the manpage in its current revision mixes Hash and Context a lot. I understand that Hash is Ruby's version of a 'dictionary' or 'object' or 'hash map', but I think it would go a long way to do away with language specific vocabulary.